src/spatial/collision.js

```diff
@@ -332,16 +325,16 @@ Crafty.c("Collision", {
      * @see .checkHits
      * @see .hit
      */
-    onHit: function (comp, callback, callbackOff) {
+    onHit: function (component, hit, noHit) {
```

Comment:
I agree, but PHPStorm highlights if function argument and phpdoc `@param` are different. May be change it in phpdoc?